tests/Integration/Behaviour/Features/Scenario/Product/update_prices_without_states.feature

```diff
@@ -0,0 +1,280 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags update-prices-without-states
+@restore-all-tables-before-feature
```

Comment:
Regarding the database strategy for clearing I think the most effective would be to use `@restore-products-before-feature` here, at the beginning of the scenario it should be enough

However since you also create taxes and taxe rules it's not enough to clear after the scenario so you could introduce a new tag in CommonFeatureContext:

```php
    /**
     * This hook can be used to flag a feature for database hard reset
     *
     * @BeforeFeature @restore-all-tables-after-feature
     */
    public static function restoreAllTablesAfterFeature()
    {
        DatabaseDump::restoreAllTables();
        require_once _PS_ROOT_DIR_ . '/config/config.inc.php';
    }
```

This way you can use `@restore-all-tables-after-feature` in this scenario to make sure the DB is correctly reset at the end of the feature. And it shouldn't have that much impact on the following feature even if it uses `@restore-products-before-feature` since the tables will already have been reset (the reset checks if any modification is in the DB)